### DISCUSSION TITLE
Saas app using Statamic

### DISCUSSION THREAD

**Discussion by bhaidar** [statamic/cms#7357] [General]
Hey community, 

I’m still discovering statamic here. 

I have a few questions if you have the time to check them out. 

- I could have a standard Laravel app and install Statamic over. I believe this is mentioned in the docs.
- I can enable user login to a custom dashboard (other than statamic) like I do in any other Laravel app without getting into issues with statamic?
- I can develop pages using InertiaJs without having any collision with Statamic? 
- Any other development with backend or frontend Laravel can be done the same way with or without Statamic without any issues? 

I appreciate your time checking my questions. 

Bill

**Comment by jackmcdade** [comment]
Yes to all!

**Reply by bhaidar** [reply]
Amazing! I’ll schedule a sample app soon and write about and share all the steps needed it might help new comers to a statamic like me :-)

**Reply by jackmcdade** [reply]
Awesome idea!
